opentelemetry-sdk/src/trace/config.rs

```diff
@@ -129,9 +129,9 @@ impl Default for Config {
                     if let Some(r) = ratio {
                         Box::new(Sampler::TraceIdRatioBased(r))
                     } else {
-                        handle_error(
-                            Error::Other(String::from(
-                                "Missing or invalid OTEL_TRACES_SAMPLER_ARG value. Falling back to default: 1.0"))
+                        otel_warn!(
+                            name: "Sampler.TraceIdRatio.InvalidArgument",
```

Comment:
the name is misleading. This is not an issue occurring in the Sampler (it affects sampler), but rather a configuration issue.

Reply:
I didn't get this - isn't this the configuration for sampler which is causing the issue? In that case, it is related to sampler.

Reply:
this is purely configuration issue, that *affects* sampler, not an issue inside sampler. Sampler's code is not even executed, so the name is misleading, as it seem to indicate some issue inside Sampler. If this log is emitted from inside a Sampler's `new()` method, then the name makes sense, but in this case, it is occurring outside Sampler.

Reply:
updated as below:
```rust
    name: "TracerProvider.Config.Sampler.InvalidArgument",
    message = "OTEL_TRACES_SAMPLER is set to 'traceidratio' but OTEL_TRACES_SAMPLER_ARG environment variable is missing or invalid. OTEL_TRACES_SAMPLER_ARG must be a valid float between 0.0 and 1.0 representing the desired sampling probability (0.0 = no traces sampled, 1.0 = all traces sampled, 0.5 = 50% of traces sampled). Falling back to default ratio: 1.0 (100% sampling)"
 );
```